docs/samples/product_recommendation_demo.ipynb

```diff
@@ -4,23 +4,13 @@
       "cell_type": "markdown",
       "metadata": {},
       "source": [
-        "# Feathr Feature Store on Azure Demo Notebook\n",
+        "# Demo Notebook: Feathr Feature Store on Azure\n",
         "\n",
-        "This notebook illustrates the use of Feature Store to create a model that predicts product ratings. It includes these steps:\n",
+        "This notebook demonstrates how Feathr Feature Store can simplify and empower your model training and inference. You will learn:\n",
         "\n",
-        "\n",
-        "This tutorial demonstrates the key capabilities of Feathr, including:\n",
-        "\n",
-        "1. Install and set up Feathr with Azure\n",
-        "2. Create shareable features with Feathr feature definition configs.\n",
-        "3. Create a training dataset via point-in-time feature join.\n",
-        "4. Compute and write features.\n",
-        "5. Train a model using these features to predict fares.\n",
-        "6. Materialize feature value to online store.\n",
-        "7. Fetch feature value in real-time from online store for online scoring.\n",
-        "\n",
-        "In this tutorial, we use Feathr Feature Store to create a model that predict users rating for a product. The feature flow is as below:\n",
```

Comment:
Can we add more flow nodes to make this flow image more align with what is demoed in notebook? For example, feature join job result is used for model training, and materialization result is used for inferencing model service.

Reply:
Does the first graph meet your expectation?

For this graph, I think we need a zoomed in view of feature materialization and that's what i am trying to provide here.

Let me know if that works or not for you.

Reply:
Talked to hangfei offline. This will be addressed in a following up PR.